packages/cli/generators/app/templates/src/application.ts.ejs

```diff
@@ -19,6 +19,21 @@ export class <%= project.applicationName %> extends BootMixin(
 export class <%= project.applicationName %> extends BootMixin(RestApplication) {
 <% } -%>
   constructor(options: ApplicationConfig = {}) {
+    options = Object.assign(
+      {
+        rest: {
+          // your application listening port number, you can change this once
+          // in production
+          port: 3000,
```

Comment:
Does it make sense now to have `port: +process.env.PORT || 3000` now?

Reply:
Yes, you are right, I forgot about the process.env.PORT support

Reply:
is the plus symbol `+process.env.PORT` intentional ?

Reply:
Yes. It converts the value to `number`.